### ISSUE TITLE
yield codepage in _extract_vba so VBA can be decoded correctly 

### ISSUE THREAD

**Issue by c1fe** [decalage2/oletools#108]
It would be very helpful if _extract_vba() yielded the codepage that it uses to decode stream names so that the codepage can be used to decode the extracted VBA as well. 

Currently it is not possible to parse VBA written with non-Western locales without decoding them correctly since the identifiers will often contain illegal characters. 

I've hacked this change locally and its been very successful. I can make a pull request with my changes if you think that would be worthwhile.

**Comment by decalage2** [comment]
Yes you're right. I need to change olevba to convert strings and macro code to Unicode for internal processing, in order to support both Python 2 and 3. And the API should either return the proper codepage for decoding, or return Unicode strings.

I'm interested in your changes if you can make a PR. I'll check if it works without breaking the current API, or if we need a new one.

Thanks!

**Comment by decalage2** [comment]
BTW, I think the code pages for stream names and VBA code can be different. There is a codepage attribute in the VBA project, which is not exposed to the API yet.
